assets/js/modules/ads/pax/services.test.js

```diff
@@ -93,6 +107,44 @@ describe( 'PAX partner services', () => {
 					).toEqual( [] );
 				} );
 			} );
+
+			describe( 'getPageViewConversionSetting', () => {
+				it( 'should hold correct value for enablePageViewConversion property', async () => {
+					const pageViewConversionSetting =
+						await services.conversionTrackingService.getPageViewConversionSetting();
+
+					expect(
+						pageViewConversionSetting.enablePageViewConversion
+					).toBeTruthy();
+				} );
+
+				it( 'should hold correct value for websitePages property', async () => {
+					const wpPagesEndpoint = new RegExp( '^/wp/v2/pages' );
+
+					fetchMock.getOnce( wpPagesEndpoint, {
+						body: mockWpRestPagesResponse,
+						status: 200,
+					} );
+
+					const pageViewConversionSetting =
+						await services.conversionTrackingService.getPageViewConversionSetting();
+
+					expect( fetchMock ).toHaveFetched( wpPagesEndpoint, {} );
```

Comment:
Let's also assert that the `per_page` was passed as expected. This is rather important since the default is 10 so we'd want to make sure this there.

To do this, we need to get the actual fetched URL (`fetchMock.lastCall( wpPagesEndpoint )`) this will return an array of the args it was called with `[ url, options ]`. 
<img width="446" alt="image" src="https://github.com/google/site-kit-wp/assets/1621608/86f9fd6a-bf52-4829-8bdc-afa768716554">

Then we have a dedicated custom matcher for making assertions on parameters (see `toMatchQueryParameters`)

Reply:
Thanks @aaemnnosttv this has been addressed in the latest commit.